resources/styles/components/concept-coach/launcher.less

```diff
@@ -1,3 +1,6 @@
+@import '../../variables';
+@import '../../mixins';
```

Comment:
I was hoping to be able to compile just the file to make sure my refactoring didn't break anything but noticed `@openstax-white` wasn't defined anywhere anyway, so I guess not; I'll remove :smile: 

I think @dak has a policy in webview for always including those files (as boilerplate) but it might just be because `webview` async loads less files?